infra/cars-secrets.yml

```diff
@@ -4,4 +4,4 @@ metadata:
   name: cars-secrets
 type: Opaque
 data:
-  KAFKA_PORT: a2Fma2Etc2VydmljZTo5MDky
\ No newline at end of file
+  KAFKA_PORT: bXktY2x1c3Rlci1rYWZrYS1ib290c3RyYXA6OTA5Mg==
```

Comment:
@vpetrovaa why do we need to store `port` in the Secret?

Reply:
@h1alexbel this secret means host+port. Change to only host and move port to deployment?

Reply:
I think we can move this information into deployment, host can be stored in the `ConfigMap`